src/components/ModuleDetailsCard/index.tsx

```diff
@@ -216,17 +217,36 @@ const ModuleDetailsCard = ({
                     justifyContent={'flex-start'}
                     flexWrap={wrap()}
                   >
-                    <Grid item flexGrow={0} xs={gridSize()}>
-                      <DeliverableStatusBadge
-                        showSimplifiedStatus={showSimplifiedStatus}
-                        status={deliverable.status || 'Not Submitted'}
-                      />
-                    </Grid>
-                    <Grid item flexGrow={0} xs={gridSize()}>
-                      <Link fontSize='16px' onClick={deliverable.onClick} style={{ textAlign: 'left' }}>
-                        {deliverable.name}
-                      </Link>
-                    </Grid>
+                    {module.name === 'Pre-screen' ? (
```

Comment:
Can we use a prop for this? I think DeliverableDetails can probably have a useButton prop.

Reply:
Sure!